Makefile

```diff
@@ -1,14 +1,21 @@
-.PHONY: all
+MAKE_VARIABLES := $(.VARIABLES)
 
 GO_VERSION=$(shell go version | cut -d ' ' -f 3 | sed -e 's/ /-/g' | sed -e 's/\//-/g' | sed -e 's/^go//g')
-GO_PLATFORM ?= $(go version | cut -d ' ' -f 4 | sed -e 's/ /-/g' | sed -e 's/\//-/g')
+GO_PLATFORM ?= $(shell go version | cut -d ' ' -f 4 | sed -e 's/ /-/g' | sed -e 's/\//-/g')
```

Comment:
I mistakenly omitted `shell` here which broke the default `GO_PLATFORM` computed from `go version`.
Brought back the `shell` call and the issue disappeared.